index.html

```diff
@@ -13,10 +13,22 @@
                 background-color:#30323d;
             }
 
-            p {
+            p.blocktext {
```

Comment:
If the p and ol share the same style, you can just do p, ol { } and delete either one of them!